src/interfaces/advanced-options.ts

```diff
@@ -1,8 +1,10 @@
 export interface AdvancedOptions {
   // A set of custom backoff strategies keyed by name.
   backoffStrategies?: {};
+  disableRun?: boolean;
```

Comment:
Maybe `disableAutoRun` to be extra explicit? :) And also can you add a quick tidbit to the Changelog :D